Configuration/StandardSequences/python/Reconstruction_Data_cff.py

```diff
@@ -41,3 +41,6 @@
 HcalRemoveAddSevLevel.AddFlag(hcalRecAlgos,"HBHESpikeNoise",12)
 
 CSCHaloData.ExpectedBX = cms.int32(3)
+
+from JetMETCorrections.Configuration.JetCorrectors_cff import ak4PFCHSResidualCorrector, ak4PFCHSL1FastL2L3ResidualCorrector, ak4PFCHSL1FastL2L3ResidualCorrectorChain
+jetCorrectorsForReco.replace(ak4PFCHSL1FastL2L3CorrectorChain, ak4PFCHSL1FastL2L3ResidualCorrectorChain)
```

Comment:
yes, residual corrections are available only in data GTs.
So, this is the only place where we can turn them on.

From RECO configuration perspective, I wish that instead dummy pass-through corrections were available in MC GT as well and we didn't have to modify configurations.